apps/src/applab/dontMarshalApi.js

```diff
@@ -180,6 +180,32 @@ export function appendItem(
   }
 }
 
+function interpreterGetValue(object, key) {
+  return getValue(object, key, true);
+}
+
+export function getValue(object, key, calledWithinInterpreter) {
+  const objectContents = calledWithinInterpreter ? object.properties : object;
+  const keyData = key.data;
+  const valueData = objectContents[keyData];
+  if (valueData === undefined) {
+    outputWarning(key + ' is not in ' + object);
+    return undefined;
+  }
```

Comment:
Updated so that users will see a warning if the key isn't in the object for getValue: 

![Screen Shot 2021-04-12 at 2 58 23 PM](https://user-images.githubusercontent.com/12300669/114446966-d4f71500-9b9f-11eb-8923-d581eccb427c.png)